geom/gdml/inc/TGDMLWrite.h

```diff
@@ -219,6 +202,7 @@ class TGDMLWrite : public TObject {
    TString GetPattAxis(Int_t divAxis, const char * pattName, TString& unit);
    Bool_t IsNullParam(Double_t parValue, TString parName, TString objName);
    void UnsetTemporaryBits(TGeoManager * geoMng);
+   void SetFltPrecision(const char *prec) { fFltPrecision = prec; }
```

Comment:
Consider std::string_view as the parameter type.

Reply:
Actually, shouldn't this just be a UInt_t to both be sturdier and more consistent with the rest of the merge request.